apps/studio/src/components/PageEditor/ComponentSelector.tsx

```diff
@@ -129,15 +128,8 @@ function ComponentSelector() {
     setSavedPageState(nextPageState)
     setDrawerState({ state: nextState })
     setCurrActiveIdx(nextPageState.content.length - 1)
+    setAddedBlockIndex(nextPageState.content.length - 1)
```

Comment:
the bug only happens when we add a new component and exit. in here. we set the state so that we know (in the editor) that this index is a block that is newly added **but doesnt exist in db**